rpi_backlight/cli.py

```diff
@@ -1,6 +1,5 @@
 from argparse import ArgumentParser
-
-from . import Backlight, __version__
+from . import Backlight, __version__, BoardType
```

Comment:
It is good practice to keep import groups separated by a newline (standard library, 3rd party, own modules) - and I also prefer some kind or ordering.

```suggestion

from . import Backlight, BoardType, __version__
```

Reply:
Thanks, I was not aware of this standard